src/app/exams/exams-add.component.ts

```diff
@@ -125,4 +125,8 @@ export class ExamsAddComponent implements OnInit {
     this.router.navigate([ this.coursesService.returnUrl ]);
   }
 
+  backToCourse() {
```

Comment:
please add `this.coursesService.reset();` if not reset it next course edit will trigger to same course